app/src/main/java/com/edulectronics/tinycircuit/Models/Scenarios/ImplementedScenarios/Scenario2.java

```diff
@@ -77,8 +78,7 @@ public boolean isCompleted(Circuit circuit) {
         Powersource powersource = new Powersource();
         Lightbulb bulb = new Lightbulb();
 
-        Connector.connect(powersource.getOutput(), bulb.getConnectionPointByIndex(0));
-        Connector.connect(powersource.getInput(), bulb.getConnectionPointByIndex(1));
+        Connector.connect(powersource.getInput(), bulb.getConnectionPointByIndex(0));
```

Comment:
Mooi, dan mag hij er van mij in, Ik heb scenario3 hierook op gebasseerd dus zal die pushen als deze gemerged is